File: razorpay_payment_links_integration/customizations/payment_entry.js
frappe.ui.form.on('Payment Entry', {
	refresh(frm){
        if(frm.doc.razorpay_payment_link){
            frm.add_custom_button(__('Resend Notification'), () => {
                frappe.call({
                    method: "razorpay_payment_links_integration.controller.resend_notification",
                    args: {"paymentLinkId": frm.doc.razorpay_payment_link},
                    callback: function(r) {
                        console.log(r);
                    }
                });
            })
        }
    }
})

File: razorpay_payment_links_integration/customizations/payment_entry.py
# import frappe
# from razorpay_payment_links_integration.controller import create_payment_link

# def before_submit(self, method=None):
#     '''
#     send the pe_doc for create payment link
#     '''
#     payment_link_doc = create_payment_link(self)
#     if(payment_link_doc.get('error')):
#         frappe.throw(
#             title='Razorpay Error',
#             msg='Error while generating Razorpay Payment Link'
#         )
#     else: 
#         self.razorpay_payment_link = payment_link_doc.get('name')
